src/org/omegat/Main.java

```diff
@@ -255,6 +256,11 @@ private static void applyConfigFile(String path) {
      * Execute standard GUI.
      */
     protected static int runGUI() {
+        ClassLoader cl = ClassLoader.getSystemClassLoader();
+        MainClassLoader mainClassLoader = (cl instanceof MainClassLoader) ? (MainClassLoader) cl : new MainClassLoader(cl);
+        PluginUtils.getThemePluginJars().forEach(mainClassLoader::add);
+        UIManager.put("ClassLoader", mainClassLoader);
```

Comment:
Here is a hack to tell UIComponents to use custom class loader.
https://github.com/openjdk/jdk/blob/6765f902505fbdd02f25b599f942437cd805cad1/src/java.desktop/share/classes/javax/swing/UIDefaults.java#L784-L790

```
        Object cl = get("ClassLoader");
        ClassLoader uiClassLoader =
            (cl != null) ? (ClassLoader)cl : target.getClass().getClassLoader();
        Class<? extends ComponentUI> uiClass = getUIClass(target.getUIClassID(), uiClassLoader);
```